crates/circuit/src/lib.rs

```diff
@@ -80,6 +81,17 @@ impl From<Clbit> for BitType {
     }
 }
 
+#[inline(always)]
+#[doc(hidden)]
+fn add_submodule<F>(m: &Bound<PyModule>, constructor: F, name: &str) -> PyResult<()>
+where
+    F: FnOnce(&Bound<PyModule>) -> PyResult<()>,
+{
+    let new_mod = PyModule::new_bound(m.py(), name)?;
+    constructor(&new_mod)?;
+    m.add_submodule(&new_mod)
+}
```

Comment:
Do we really need a submodule for this? I'm totally fine if you wanted to add this as `qiskit._accelerate.circuit.circuit_to_dag` and eventually as `dag_to_circuit()` too. I don't think we're going to have any other converter functions in rust in the near future. But if we did we can address this then.

Reply:
I ended up adding it through `py_ext` and the new path is exposed as `qiskit._accelerate.converters` to avoid adding all the extra code.